pom.xml

```diff
@@ -397,6 +397,7 @@
         </configuration>
       </plugin>
       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
Hello @SingingBush ,
Thank you for your PR.
Please keep your chanes to the minimum and focused. These types of changes are not required.

Reply:
no problem, fixing now